src/Angor/Client/Pages/Invest.razor

```diff
@@ -1272,38 +1207,58 @@ else
         return false;
     }
 
-    private void ShowChatModal()
+    private void ShowMessageModal()
```

Comment:
The naming I prefer we use is `ShowMessageWithPassword` and bellow instead of `InitializeMessaging` call it `ShowMessage`